kubernetes/windows/hostlogs/main.ps1

```diff
@@ -86,13 +86,19 @@ Set-EnvironmentVariables
 Start-FileSystemWatcher
 
 if(Get-GenevaEnabled){
-    Invoke-Expression ".\opt\genevamonitoringagent\genevamonitoringagent\Monitoring\Agent\MonAgentLauncher.exe -useenv"
+    Write-Host "Starting Windows AMA in 1P Mode"
+
+    Start-Job -Name "WindowsHostLogsJob" -ScriptBlock { 
+        Start-Process -NoNewWindow -FilePath ".\opt\genevamonitoringagent\genevamonitoringagent\Monitoring\Agent\MonAgentLauncher.exe" -ArgumentList @("-useenv")
+    }
+
+    [System.Environment]::SetEnvironmentVariable("HOSTLOGS_MA_STARTED", "true", "process")
```

Comment:
I thought I tested that, but I think I need to just exec into the container and the host to check.

Reply:
As mentioned in my [other comment](https://github.com/microsoft/Docker-Provider/pull/1024/files#r1270059764), I think "user" is the best level to put this at. Until we have containerd 1.7 we can set it to false in setup.ps1 to make sure it's cleared before restarting the container

Reply:
From what I am seeing this this environment variable isn't seen by the livenessprobe 

https://github.com/microsoft/Docker-Provider/blob/cae75f9854df1842b07886a7bbb3a5ee065945eb/kubernetes/windows/hostlogs/main.ps1#L67

Reply:
As a process env var it will get cleaned up off the container when it is no longer there. As a machine env var then it would stay on the host even if the container crashes/restartes/vacates.

Reply:
Starting up MonAgent with placeholder values until config map is applied